mono/metadata/threads.c

```diff
@@ -6350,8 +6326,14 @@ summarizer_supervisor_start (SummarizerSupervisorState *state)
 	g_assert_not_reached ();
 #endif
 
-	if (pid != 0)
+	if (pid != 0) {
 		state->supervisor_pid = pid;
+		char pid_str[20]; // pid is a uint64_t, 20 digits max in decimal form
```

Comment:
I'd be simpler to just say `65` instead of knowing how many decimal digits a 64bit number can expand to.